### ISSUE TITLE
Hardware: Pair with Bluetooth, not be dependent on Wi-fi

### ISSUE THREAD

**Issue by adamcohenhillel** [adamcohenhillel/AdDeus#8]
Currently it is using Wi-Fi, which makes it not-so mobile (you can technically connect it to your phone hot-spot).

An alternative approach would either be:
  - [ ] Bluetooth pairing with the mobile device
  - [ ] Sdd a 4G card that will allow it to be completely independent

**Comment by Krupskis** [comment]
@adamcohenhillel I am working on this for esp32

**Comment by adamcohenhillel** [comment]
@Krupskis that is great, assigned the ticket to you then.

Have you been able to run on the esp32 at the end? just started playing with mine yday

**Comment by Krupskis** [comment]
Yes, got the bluetooth and mic working separately. Trying to put it all together. The mic is weird tho, I get a bit pitched sound.

**Comment by adamcohenhillel** [comment]
@Krupskis anything I can help with? if you want, you can share the code and I will try to debug the sound stuff

**Comment by tomasmcm** [comment]
This is a super cool project! 
Looking at the bluetooth implementation, it seems the plan is to live stream audio from the device to a phone over bluetooth. So, how is that different from just using a simple bluetooth microphone ($3/$4 on aliexpress)?
I think the main selling point of Pendant and Tag is that they work by themselves, and then periodically upload to the cloud.
Since most microcontrollers won't have a lot of memory, you could use an SD card to store the audio, and when bluetooth or wifi is available upload it to supabase.
wdyt?
